website/algolia-lockfile.txt

```diff
@@ -0,0 +1,1144 @@
+[
```

Comment:
why txt? it's json file

Reply:
Idk, following the convention like `route-lockfile.txt`, I would personally prefer to call it `algolia.lock`, wdyt?

Reply:
I prefer JSON as it's JSON content and when you'll open it in VSCode you'll get syntax highlighting too